openapi/kas-fleet-manager.yaml

```
@@ -1966,4 +1969,4 @@ components:
           for cluster id: 1g5d88q0lrcdv4g7alb7slfgnj3dhbsj%!(EXTRA *errors.Error=identifier
           is '404', code is 'CLUSTERS-MGMT-404' and operation identifier is '1g5or50viu07oealuehrkc26dgftj1ac':
           Cluster '1g5d88q0lrcdv4g7alb7slfgnj3dhbsj' not found)"
-        operation_id: "1iYTsWry6nsqb2sNmFj5bXpD7Ca"
+        operation_id: "1iYTsWry6nsqb2sNmFj5bXpD7Ca"
```

```suggestion
        operation_id: "1iYTsWry6nsqb2sNmFj5bXpD7Ca"

```
nit adding newline at end of file